src/app/fdctl/run/tiles/fd_net.c

```diff
@@ -271,6 +272,27 @@ net_rx_aio_send( void *                    _ctx,
   return FD_AIO_SUCCESS;
 }
 
+static void
+metrics_write( void * _ctx ) {
+  fd_net_ctx_t * ctx = (fd_net_ctx_t *)_ctx;
+
+  ulong rx_cnt = ctx->xsk_aio[ 0 ]->metrics.rx_cnt;
+  ulong rx_sz  = ctx->xsk_aio[ 0 ]->metrics.rx_sz;
+  ulong tx_cnt = ctx->xsk_aio[ 0 ]->metrics.tx_cnt;
+  ulong tx_sz  = ctx->xsk_aio[ 0 ]->metrics.tx_sz;
+  if( FD_LIKELY( ctx->xsk_aio[ 1 ] ) ) {
+    rx_cnt += ctx->xsk_aio[ 1 ]->metrics.rx_cnt;
+    rx_sz  += ctx->xsk_aio[ 1 ]->metrics.rx_sz;
+    tx_cnt += ctx->xsk_aio[ 1 ]->metrics.tx_cnt;
+    tx_sz  += ctx->xsk_aio[ 1 ]->metrics.tx_sz;
+  }
```

Comment:
Why are there two xsk_aio instances? Is one for localhost?

Reply:
yup